lib/shared/util.ts

```diff
@@ -0,0 +1,13 @@
+const nodeUtil = require("util");
+const _indexOf = require("lodash.indexof");
+const _sortBy = require("lodash.sortby");
```

Comment:
Do we need to? By any means underscore notation is associated with private access so I think it is misleading here.